### ISSUE TITLE
Identity_loss

### ISSUE THREAD

**Issue by Charlie1994** [feay1234/CARA#1]
Hi there,
I am trying to run the code provided, but I find that the `identity_loss` is not defined in the statement `self.model.compile(optimizer=Adam(), loss=identity_loss)` . Could you please fix this issue?

Thanks,
Charles

**Comment by mindis** [comment]
same issue

**Comment by mindis** [comment]
def identity_loss(y_true, y_pred):
    return K.mean(y_pred - 0 * y_true)

**Comment by feay1234** [comment]
Thanks @Charlie1994 and @mindis  for you guys' feedback. 

I have fixed the bug.
